lib/rules/web/admin_console/4.8/deployment.xyaml

```diff
@@ -149,4 +149,31 @@ select_replica_count_and_selector:
   params:
     text: Replica count and selector
   action: click_checkbox_input_by_text
-
+check_workloads_metrics:
```

Comment:
We can add `action: click_metrics_tab` into `check_workloads_metrics` 
```
check_workloads_metrics:
  action: click_metrics_tab
  xxxxxxx
```

Reply:
Done